pkg/scheduler/scheduler_test.go

```diff
@@ -191,9 +190,6 @@ func TestSchedule(t *testing.T) {
 		// additional*Queues can hold any extra queues needed by the tc
 		additionalClusterQueues []kueue.ClusterQueue
 		additionalLocalQueues   []kueue.LocalQueue
-
-		// enable partial admission
-		enablePartialAdmission bool
```

Comment:
Yes, we need to keep this knob. We just have it enabled by default.

Reply:
The tests if needed could still disable the feature gate,  `enablePartialAdmission` is no longer needed because by not setting the `MinCount` you essentially opt out of partial admission.

Reply:
We still need tests for cases that set MinCount, but the feature is disabled.

Feature gates are a tool for administrator to disable a feature, even if end users are trying to use it.

Reply:
added one test for that